src/Repository/StatsRepository.php

```diff
@@ -105,4 +105,44 @@ protected function getStartDate(array $values): array
     {
         return array_map(fn ($val) => ['year' => $val['year'], 'month' => $val['month']], $values);
     }
+
+    protected function prepareContentReturn(array $entries, array $comments, array $posts, array $replies): array
+    {
+        $startDate = $this->sort(
+            array_merge(
+                $this->getStartDate($entries),
+                $this->getStartDate($comments),
+                $this->getStartDate($posts),
+                $this->getStartDate($replies)
+            )
+        );
+
+        if (empty($startDate)) {
+            return [
+                'entries' => [],
+                'comments' => [],
+                'posts' => [],
+                'replies' => [],
+            ];
+        }
```

Comment:
Validate $startDate before continue. Also check if `$startDate[0]` exists.

Reply:
`$startDate` is checked for being empty just above. I'll add the check whether `year` and `month` exist

Reply:
ah yes sorry my bad.